### PR TITLE
Boards: Add Raspberry Pi 3

### PR DESCRIPTION
**Pull request by tombettany**
Add overclocking settings for the Raspberry Pi 3.

cc @radujipa @alex5imon @Ealdwulf

### CODE REVIEW

File: kano_settings/system/boards/rpi_3.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#
+# Board specific settings for Raspberry Pi 3
+#
+
+DEFAULT_CONFIG = {
+    'Overclocking': 'Standard'
+}
+
+CLOCKING = {
+    'modes': ['Standard'],
```
Comment (radujipa) [inline kano_settings/system/boards/rpi_3.py]: So no overclocking for now?

File: kano_settings/system/boards/rpi_3.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#
+# rpi_2.py
```
Comment (Ealdwulf) [inline kano_settings/system/boards/rpi_3.py]: Name needs updating...

**Comment by Ealdwulf** [comment]
Apart from tiny comment note above, looks good :+1

File: kano_settings/system/boards/rpi_3.py (hunk 1 above)
Comment (Ealdwulf) [inline kano_settings/system/boards/rpi_3.py]: We asked the pi foundation, they said no.

Comment (tombettany) [inline kano_settings/system/boards/rpi_3.py]: Nope

File: kano_settings/system/boards/rpi_3.py (hunk 2 above)
Comment (tombettany) [inline kano_settings/system/boards/rpi_3.py]: Good spot

**Comment by tombettany** [comment]
Merging this one then.

**Pull request merged**